app/routes/surveys/components/SubmissionEditor.js

```diff
@@ -0,0 +1,140 @@
+                  )
+                  .filter(option => option);
+
+          return {
+            ...answer,
+            question: question.id,
+            selectedOptions,
+            answerText: answer.answerText || ''
+          };
+        })
+        .filter(answer => answer)
+    };
+
+    return submitFunction(toSubmit);
+  };
+
+  if (fetching || !survey || !survey.event || !survey.event.cover) {
+    return <LoadingIndicator loading />;
+  }
+
+  return (
+    <Content className={styles.surveyDetail} banner={survey.event.cover}>
+      <ContentHeader>{survey.title}</ContentHeader>
+
+      <div className={styles.surveyTime}>
+        Spørreundersøkelse for arrangementet{' '}
+        <Link to={`/surveys/${survey.event.id}`}>{survey.event.title}</Link>
+      </div>
+
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <ul className={styles.detailQuestions}>
+          {(survey.questions || []).map((question, i) => (
+            <li key={question.id}>
+              <h3 className={styles.questionTextDetail}>
+                {question.questionText}
+                {question.mandatory && (
+                  <span className={styles.mandatory}> *</span>
+                )}
+              </h3>
+
+              {question.questionType === 3 ? (
+                <Field
+                  component={TextArea.Field}
+                  placeholder="Skriv her..."
+                  name={`answers[${i}].answerText`}
+                  className={styles.freeText}
+                />
+              ) : (
+                <ul className={styles.detailOptions}>
+                  {(question.options || []).map((option, j) => (
+                    <li key={option.id}>
+                      {question.questionType === 1 ? (
+                        <Field
+                          component={RadioButton.Field}
+                          label={option.optionText}
+                          name={`answers[${i}].selectedOptions[${i}]`}
+                          inputValue={String(option.id)}
+                          className={styles.formOption}
+                        />
+                      ) : (
+                        <Field
+                          component={CheckBox.Field}
+                          label={option.optionText}
+                          name={`answers[${i}].selectedOptions[${j}]`}
+                          className={styles.formOption}
+                        />
+                      )}
+                    </li>
+                  ))}
+                </ul>
+              )}
+            </li>
+          ))}
+        </ul>
+
+        <div className={styles.clear} />
+        <Button className={styles.submit} disabled={submitting} submit>
+          Lagre
+        </Button>
+      </form>
+    </Content>
+  );
+};
+
+const validate = createValidator({
+  user: [required()]
+});
+
+export default reduxForm({
+  form: 'submissionEditor',
+  validate,
+  enableReinitialize: true
```

Comment:
Is there a reason you use this? ref. #1015

Reply:
Nope, copypasta. Will remove.

Reply:
You can try to refresh while in this route, to see if that still works.